### PR TITLE
Create eip155-42069

### PR DESCRIPTION
**Pull request by stealthgerbil**

### CODE REVIEW

File: _data/chains/eip155-42069 [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    "chain": "42069",
+    "network": "mainnet",
+    "rpc": [],
+    "faucets": [],
+    "nativeCurrency": {
+      "name": "pegglecoin",
+      "symbol": "peggle",
+      "decimals": 18
+    },
+    "infoURL": "https://pegglecoin.com",
```
Comment (ligi) [inline _data/chains/eip155-42069:13]: this site seems to not exist

**Review by ligi — CHANGES_REQUESTED** [review]

**Review by stealthgerbil — COMMENTED** [review]
updated site, it was wrong

File: _data/chains/eip155-42069 (hunk 1 above)
Comment (stealthgerbil) [reply _data/chains/eip155-42069]: Sorry was wrong site. Set it to our communities homepage.

**Comment by ligi** [comment]
I dropped a mail to the email mentioned on the website and waiting for reply. Do not really see a connection between the website and the chain

**Comment by stealthgerbil** [comment]
> I dropped a mail to the email mentioned on the website and waiting for reply. Do not really see a connection between the website and the chain

Thank you, I let the person at badmin@teampeggle.com know. We are setting up a chain for our internet community to use for fun projects and maybe integrated with game servers. We have not added anything to our website yet because we are trying to get everything set up first so it is easy for everyone to use.  Thank you for your help, we really appreciate it so much.

**Comment by ligi** [comment]
cool - got the mail 
Now we only need to make the CI happy. Currently it is unhappy as another chain uses 420 as short name (_data/chains/eip155-2020.json)  - either rename or coordinate with them to rename. ShortName can only be used once - this is a condition by TREZOR

**Review by stealthgerbil — COMMENTED** [review]
Updated shortname so it is unique

**Comment by stealthgerbil** [comment]
I have updated the shortname of the chain, thank you so much for your patience with this.
________________________________

**Comment by ligi** [comment]
```
Exception in thread "main" org.ethereum.lists.chains.model.ExtensionMustBeJSON: filename extension must be json
        at org.ethereum.lists.chains.MainKt.checkChain(Main.kt:39)
        at org.ethereum.lists.chains.MainKt.main(Main.kt:17)
```

**Review by stealthgerbil — COMMENTED** [review]
it is a json file now, I am really sorry about that.

**Review by ligi — APPROVED** [review]

**Comment by ligi** [comment]
no need to be sorry - CI is happy now and PR is merged

**Comment by stealthgerbil** [comment]
Thank you so much, you are the best!
________________________________

**Pull request merged**